lib/v2/sigsac/ccs.js

```diff
@@ -0,0 +1,77 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const url = 'https://www.sigsac.org/';
+// https://www.sigsac.org/ccs/CCS2022/program/accepted-papers.html
+
+module.exports = async (ctx) => {
+    const items = [];
+    let promises = [];
+    const urls = [];
+
+    const last = new Date().getFullYear() - 2000 + 1;
+
+    // Get accpeted papers url from index page
+    for (let index = 20; index < last; index++) {
+        promises.push(
+            got
+                .get(`${url}/ccs/CCS20${index}/`)
+                .then((response) => response.body)
+                .catch((e) => {
+                    // ignore 404
+                    /* deepscan-disable */
+                    // eslint-disable-next-line no-use-before-define
+                    e;
+                })
+        );
+    }
+
+    let urlArray = await Promise.all(promises).then((res) => res);
+    promises = [];
+    urlArray = urlArray.filter((element) => element !== undefined);
+    urlArray.forEach((res) => {
+        const $ = cheerio.load(res);
+        urls.push($('a:contains("Accepted Papers")').attr('href'));
+    });
+
+    // Format urls
+    for (let index = 20; index < last; index++) {
+        urls[index - 20] = new URL(urls[index - 20], `${url}/ccs/CCS20${index}/`);
+    }
```

Comment:
Similary https://github.com/DIYgod/RSSHub/pull/11498#discussion_r1056969416
```suggestion
    const last = new Date().getFullYear() + 1;
    const yearList = Array.from({ length: last - 2020 }, (_, v) => `${url}ccs/CCS${v + 2020}/`);
    const yearResponses = await got.all(yearList.map((url) => got(url)));

    const urlList = yearResponses.map((response) => {
        const $ = cheerio.load(response.data);
        return new URL($('a:contains("Accepted Papers")').attr('href'), response.url).href;
    });

    const responses = await got.all(urlList.map((url) => got(url)));
```